### ISSUE TITLE
Taiko does not show the correct message for readonly text box

### ISSUE THREAD

**Issue by Debashis9012** [getgauge/taiko#613]
### Expected behavior
User can not perform write action on readonly text. 

### Actual behavior
Taiko shows the successful write operation on readonly text. 
```
> write("hello",textBox(toRightOf("First name:")))
 ✔ Wrote hello into the text field To right of First name:
```

### Steps to reproduce
1. Refer the below html snippet
```
<html>
<body>

<form action="/action_page.php">
First name: <input type="text" name="FirstName" readonly value="Mickey"><br>
Last name: <input type="text" name="LastName" value="Mouse"><br>

</body>
</html>
```
2. Follow the below steps:
  ```
   const { openBrowser, goto, toRightOf, textBox, write, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await goto("file:///path of html file");
        await write("hello",textBox(toRightOf("First name:")));
    } catch (e) {
        console.error(e);
    } finally {
        await closeBrowser();
    }
})();
```

### Versions
```
Version: 0.8.0 (Chromium: 76.0.3803.0) 7493c6afa8565c8482f8f01ada43c32dd1185627
```

**Comment by nehashri** [comment]
Few considerations for this issue,
- If the element currently focused is readonly, the write api should throw an error.
- If the selector returns more than one element to the write api, we should check for the first element which is not readonly and write to that element.

**Comment by Apoorva-GA** [comment]
Fix available in commit : 5fb590f

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed as per the two above considerations.
Commit version: 5fb590f
